spec.html

```diff
@@ -1918,8 +1918,8 @@ <h1>BigInt::sameValueZero ( _x_, _y_ )</h1>
         <emu-clause id="sec-binaryand" aoid="BinaryAnd">
           <h1>BinaryAnd ( _x_, _y_ )</h1>
           <emu-alg>
-            1. Assert: _x_ is 0 or 1.
-            1. Assert: _y_ is 0 or 1.
+            1. Assert: _x_ is +0 or 1.
+            1. Assert: _y_ is +0 or 1.
```

Comment:
The arguments to `BinaryAnd`, `BinaryOr`, and `BinaryXor` are probably BigInt values, not Numbers (see issue #1723), so the tests should probably be `is *0n* or *1n*`.